iothub/service/src/Registry/Models/ImportJobProperties.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains properties of an import job.
+    /// </summary>
+    /// <seealso href="https://docs.microsoft.com/rest/api/iothub/service/createimportexportjob"/>
+    public class ImportJobProperties : JobProperties
+    {
+        /// <summary>
+        /// Serialization constructor.
+        /// </summary>
+        internal ImportJobProperties()
+        { }
+
+        /// <summary>
+        /// Creates an instance of this class for the import job.
+        /// </summary>
+        /// <seealso cref="DevicesClient.ImportAsync(ImportJobProperties, System.Threading.CancellationToken)"/>
+        /// <param name="inputBlobContainerUri">URI to a blob container that contains registry data to sync.</param>
+        /// <param name="outputBlobContainerUri">URI to a blob container, used to output the status of the job and the results.
+        /// If not specified, the input blob container will be used.</param>
+        public ImportJobProperties(Uri inputBlobContainerUri, Uri outputBlobContainerUri = default)
+        {
+            Type = JobType.ImportDevices;
```

Comment:
I can't leave a common on the base class property, but it should be changed to `protected` instead of internal. Or maybe `protected internal` if we set it in unit tests.